go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/Azure/azure-storage-queue-go v0.0.0-20181215014128-6ed74e755687
 	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
 	github.com/Azure/go-autorest/autorest v0.9.3
+	github.com/Azure/go-autorest/autorest/adal v0.8.1
```

Comment:
Please also add this to `docs/LICENSE_OF_DEPENDENCIES.md`. Currently it seems like our check-script is on strike, so watch the output with care. :-)

Reply:
Ok seems like it is covered by `github.com/Azure/go-autorest/autorest` already. Please mark as resolved.